power_grid_model_c/power_grid_model/include/power_grid_model/math_solver/iterative_linear_se_solver.hpp

```diff
@@ -117,12 +118,12 @@ template <bool sym> class IterativeLinearSESolver {
             sub_timer = Timer(calculation_info, 2225, "Solve sparse linear equation (pre-factorized)");
             sparse_solver_.solve_with_prefactorized_matrix(data_gain_, perm_, x_rhs_, x_rhs_);
             sub_timer = Timer(calculation_info, 2226, "Iterate unknown");
-            max_dev = iterate_unknown(output.u, measured_values.has_angle_measurement());
+            max_dev = iterate_unknown(output.u, measured_values.has_angle());
```

Comment:
Why is this renamed to `has_angle`, when the function is to check if it has an angle measurements (implementation: `constexpr bool has_angle() const { return n_voltage_angle_measurements_ > 0; }`)

Reply:
to keep the similarity with `has_voltage`. we can also bump both to `has_*_measurement`

Reply:
Both are fine by me, so we can keep it as is. Good idea to keep similarity :)